packages/edit-site/src/components/navigation-sidebar/navigation-panel/index.js

```diff
@@ -100,21 +69,48 @@ const NavigationPanel = ( { isOpen } ) => {
 					</div>
 				</div>
 				<div className="edit-site-navigation-panel__scroll-container">
-					<Navigation
-						activeItem={ activeItem }
-						activeMenu={ activeMenu }
-						onActivateMenu={ setActive }
-					>
-						{ activeMenu === MENU_ROOT && (
-							<MainDashboardButton.Slot>
-								<NavigationBackButton
-									backButtonLabel={ __( 'Dashboard' ) }
-									className="edit-site-navigation-panel__back-to-dashboard"
-									href="index.php"
+					<Navigation activeItem={ activeTemplateType }>
+						<MainDashboardButton.Slot>
+							<NavigationBackButton
+								backButtonLabel={ __( 'Dashboard' ) }
+								className="edit-site-navigation-panel__back-to-dashboard"
+								href="index.php"
+							/>
+						</MainDashboardButton.Slot>
+
+						<NavigationMenu>
+							<NavigationGroup title={ __( 'Editor' ) }>
+								<NavigationItem
+									title={ __( 'Site' ) }
+									href={ addQueryArgs( '', {
+										page: 'gutenberg-edit-site',
+									} ) }
+								/>
+								<NavigationItem
+									title={ __( 'Styles' ) }
+									href={ addQueryArgs( '', {
+										page: 'gutenberg-edit-site',
+										style: 'open',
```

Comment:
Just double-checked, and I think this is definitely a typo that's causing it not to work:
```suggestion
										styles: 'open',
```